packages/pyright-internal/src/localization/package.nls.zh-cn.json

```diff
@@ -48,25 +48,26 @@
         "assignmentTargetExpr": "表达式不能被赋值",
         "asyncNotInAsyncFunction": "异步函数之外不允许使用 \"async for/with/...\"",
         "awaitIllegal": "使用 `await` 需要 Python 3.5 或更高版本",
-        "awaitNotAllowed": "类型注解中不能使用 `await`",
+        "awaitNotAllowed": "类型表达式中不能使用 `await`",
         "awaitNotInAsync": "仅允许在异步函数内使用 `await`",
-        "backticksIllegal": "Python 3.x 中不支持使用反引号包围的表达式；请改用 repr(...)",
+        "backticksIllegal": "Python 3.x 中不支持使用反引号包围的表达式；请改用 `repr(...)`",
         "baseClassCircular": "类不能继承自身",
-        "baseClassFinal": "基类 \"{type}\" 被 `@final` 装饰，因此不能被子类继承",
+        "baseClassFinal": "基类 \"{type}\" 已被 `@final` 装饰，因此不能被子类继承",
         "baseClassIncompatible": "{type} 的基类之间不相互兼容",
         "baseClassInvalid": "类的参数必须是基类",
         "baseClassAny": "`Any` 类型的基类覆盖了子类的类型",
         "baseClassMethodTypeIncompatible": "方法 \"{name}\" 的定义与 \"{classType}\" 类的基类不兼容",
         "baseClassUnknown": "未知类型的基类覆盖了子类的类型",
         "baseClassVariableTypeIncompatible": "变量 \"{name}\" 的定义与 \"{classType}\" 类的基类不兼容",
-        "binaryOperationNotAllowed": "类型注解中不允许使用二元运算符",
+        "binaryOperationNotAllowed": "类型表达式中不允许使用二元运算符",
         "bindTypeMismatch": "无法绑定方法 \"{methodName}\"，因为 \"{type}\" 不能赋值给 \"{paramName}\" 参数",
+        "breakInExceptionGroup": "不允许在 `except*` 语句块中使用 `break`",
         "breakOutsideLoop": "`break` 只能在循环中使用",
         "callableExtraArgs": "`Callable` 只能传入两个类型参数",
         "callableFirstArg": "此处应为参数类型列表或 `...`",
         "callableNotInstantiable": "无法实例化 \"{type}\" 类型",
         "callableSecondArg": "返回类型应为 `Callable` 的第二个类型参数",
-        "casePatternIsIrrefutable": "只有最后一个 `case` 语句允许使用模式 `_`",
+        "casePatternIsIrrefutable": "只有最后一个 `case` 语句允许使用 `_` 模式",
```

Comment:
the english message for this is "Irrefutable pattern is allowed only for the last case statement", because i think any variable name is considered "irrefutable" (ie. always matches):

Code sample in [basedpyright playground](https://basedpyright.com/?pythonVersion=3.13&typeCheckingMode=standard&code=CYUwZgBGD20BQDcCGAbAriAXBAlgOwBcBKTAWACgIqIBbJAgYwAsJl0sLquIGkBnEBCRlK3MQAcATvgJwkRTmJ79BAJhFLuAOh1A)

```python
def foo(value: int):
    match value:
        case a: # error: Irrefutable pattern is allowed only for the last case statement
            print(a)
        case 2:
            ...
```

is there a chinese word for "irrefutable" that would make sense to use here instead?

Reply:
happy to merge this now if you would rather address it in a follow-up PR, since this part of the message is unchanged

Reply:
I looked up the Cambridge dictionary and noticed that the explanation of 'irrefutable' is "impossible to prove wrong", in our context it may refer "impossible not to match" or "always match", so I think it would be corrected into "必定匹配（的）".

Reply:
I asked others and got other similar translations like "通配", "兜底" and so on. "通配" would be a choice as well, and "兜底" tends to be used in spoken language.

Reply:
After discussion, we decide to use "通配" to correct the translation. If there are better expressions, it will be modified again.